app/src/main/java/org/wikipedia/views/ImageTitleDescriptionView.kt

```diff
@@ -0,0 +1,88 @@
+package org.wikipedia.views
+
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.Gravity.CENTER_HORIZONTAL
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import androidx.core.content.ContextCompat
+import kotlinx.android.synthetic.main.view_image_title_description.view.*
+import org.wikipedia.R
+import org.wikipedia.util.DimenUtil
+
+internal class ImageTitleDescriptionView @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyle: Int = 0) : LinearLayout(context, attrs, defStyle) {
+
+    init {
+        View.inflate(context, R.layout.view_image_title_description, this)
+        layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+        circularProgressBar.setCurrentProgress(80.0)
+    }
+
+    fun setTitle(titleText: String) {
+        title.text = titleText
+    }
+
+    fun setTitleTextSize(textSize: Float) {
+        title.setTextSize(TypedValue.COMPLEX_UNIT_SP, textSize)
+    }
+
+    fun setCaseForTitle(isAllCaps: Boolean) {
+        title.isAllCaps = isAllCaps
+    }
+
+    fun setDescription(descriptionText: String) {
+        description.text = descriptionText
```

Comment:
Should we make the first letter in uppercase?

Reply:
This is not to update a wiki description... it is just string created by us, so i think we don't need to.

Reply:
I know. When you look at the [Zeplin ](https://app.zeplin.io/project/57a120b91998d8977642a238/screen/5d6fe672683e59b8bec38843), you'll find that the Contributions which is an uppercase on "C".

Reply:
Corrected that. Thanks.